### ISSUE TITLE
Persistent Image Display

### ISSUE THREAD

**Issue by Blann** [AntonyCorbett/OnlyM#47]
Idea in brief:

1. Display image
2. Switch directly to video
3. At video conclusion, automatically revert to previous image

This would enable the use of a certain image as a backdrop between video elements of a presentation. We had an occasion recently where this would have been ideal.

This behaviour could perhaps be an optional setting.

**Comment by AntonyCorbett** [comment]
A similar requirement is to specify a 'fallback' image when displaying _any_ other media type, not just videos. So we could have the ability to show an image in 'fallback' mode, which image is then displayed whenever no other media is shown. The media item corresponding to the fallback image should be easily recognised and manually closed. The fallback image must be closed before another fallback image is selected.

**Comment by Blann** [comment]
So, when enabled, it would function in much the same way as the background image in SoundBox, but controlled within the UI. I think that’d be good, as long as there’s a clear enough indication when a backdrop is active so as not to inadvertently interfere with JWL (for those that use the two concurrently).

**Comment by LeeOverton** [comment]
This is something I would like to see. Currently looking at away to display the Circuit Assembly theme as a permanent background image in OnlyM. I know i can do this in Soundbox but I like the simpler interface in OnlyM. As we hire a venue for the CA we take in a laptop and try to keep things as simple as possible. In this setting JWLibrary is not used for playback of any media

**Comment by GrahamR-AU** [comment]
@LeeOverton 
My solution at our last Assembly was to set the Assembly theme image as the desktop background.

**Comment by LeeOverton** [comment]
Hi @GrahamR-AU 
Thanks for the input. I was going through options again the other evening as we have next CA in March. I have now set the Permanent background option. I then select the slide for the Assembly theme and display this. When playing MP3 prelude files the theme continues to display. When I play a video song  file at its conclusion it switches back to a plain black background. I then just click on the assembly theme slide and it shows once again. It would be nice however if you could create your own permanent background as you could in Soundbox.
Thanks again
Lee

**Comment by GrahamR-AU** [comment]
@LeeOverton to my way of thinking my solution above is as good as being able to "create your own permanent background as you could in Soundbox", perhaps even better.

**Comment by LvDaniel** [comment]
I'm planning to use this at Memorial if will be ready by then.
We want to display a generic image (not the yeartext) during the meeting. 
Also during the memorial talk we'll display some images and the Bible verses (OnlyM slides). To be more easy (not to complicate things) I'm thinking also to play the songs in OnlyM (not from JWL).
If this won't be ready, I think the only option is to use the image as a desktop background (or to use SoundBox).

**Comment by Blann** [comment]
I think, if you’re not using JWL at the same time, the desktop background method works well enough. Just need to remember to remove it afterwards! Where this feature would be useful is if you do have JWL open.

**Comment by LvDaniel** [comment]
It works, but the transition from desktop image to OnlyM image is not so smooth as I'm using fade in/fade out. In SoundBox things were better from this point of view.

**Comment by Blann** [comment]
True – I suppose I should qualify that by saying that it works well enough if the majority of the background image is black and you use the fastest fade speed. The transition under those circumstances looks similar to the internal JWL transition to the untrained eye – it’s certainly not distracting in any way. 

I think we all agree that SoundBox was ideal and we’ve been trying to get as close to that experience with the Only* apps as possible, at least from an audience point of view, if not operation. I guess I’ve accepted some compromises are inevitable, but I think they’re acceptable as long as they don’t unduly detract from the quality of presentation. Appreciate not everyone will take this view though, and it’s up to Antony ultimately!

**Comment by LvDaniel** [comment]
Antony made a pretty good job and we appreciate all his efforts for transitioning from SoundBox to OnlyX apps. I'm pleased with the current OnlyM options, but this is a nice to have in case doesn't complicate things and it's easy to implement.

**Comment by tenisak** [comment]
> So we could have the ability to show an image in 'fallback' mode, which image is then displayed whenever no other media is shown.

Now that Zoom is used a lot, 'fallback image' feature would be very useful.

**Comment by timjobutler** [comment]
@tenisak Yes, this would be very helpful when using OnlyM with Zoom. Our congregation uses Yeartext image from JW library as the "persistent" image (for prayers, pre-meeting) and it would definitely make the sharing function in Zoom more smoother.

**Comment by timjobutler** [comment]
@AntonyCorbett How much work is involved is making this feature possible?
